app/src/main/res/layout/listitem_weapon.xml

```diff
@@ -34,9 +39,10 @@
         <!-- These should be removed from this layout and added at runtime -->
 
         <com.gatheringhallstudios.mhworlddatabase.components.CompactStatCell
+            android:id="@+id/attack_value"
             android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            app:iconSrc="@drawable/ic_ui_question_mark_white"
+            android:layout_height="24dp"
```

Comment:
Make this `row_height_xsmall` in dimens.xml. We don't have this size yet because this is the first time we're using it, but we'll probably use it more in the future.